azurerm/resource_arm_sql_administrator.go

```diff
@@ -0,0 +1,143 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/services/sql/mgmt/2015-05-01-preview/sql"
+	"github.com/hashicorp/terraform/helper/schema"
+	uuid "github.com/satori/go.uuid"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+	//"github.com/hashicorp/terraform/helper/validation"
+	//"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/response"
+)
+
+func resourceArmSqlAdministrator() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmSqlAdministratorCreateUpdate,
+		Read:   resourceArmSqlAdministratorRead,
+		Update: resourceArmSqlAdministratorCreateUpdate,
+		Delete: resourceArmSqlAdministratorDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
```

Comment:
given we've supported import - can we add an import test for this resource? here's an example of how we do that https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/import_arm_mysql_server_test.go

Reply:
I think I've implemented this. i'm getting a "filename too long" error, which makes it tricky to test this is working at all

Reply:
no worries, I can kick off the tests and take a look. Out of interest (I'm assuming you're running on Windows) - do you have the unlimited filename length Windows feature enabled?

Reply:
I hadn't & I have now, but it seems VS Code still complains.